drivers/backup/auth.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
rebase with latest changes and use pkg/log

Reply:
Addressed